gradle/publish.gradle

```diff
@@ -36,22 +36,22 @@ publishing {
         }
     }
 
-    repositories {
-        maven {
-            def releasesRepoUrl = "https://oss.sonatype.org/service/local/staging/deploy/maven2/"
-            def snapshotsRepoUrl = "https://oss.sonatype.org/content/repositories/snapshots/"
-            url = version.toString().endsWith('SNAPSHOT') ? snapshotsRepoUrl : releasesRepoUrl
 
-            credentials {
-                username = project.findProperty('sonatype_username')
-                password = project.findProperty('sonatype_password')
+    repositories {
+        if (version.version.isSignificant()) {
```

Comment:
You will probably need `version.toString().endsWith("SNAPSHOT")`, unless you are using reckon for your versions.

Reply:
Okay, so you are using reckon for versions :smile: